website/package.json

```
@@ -18,19 +18,19 @@
     "generate-changelog-markdown-files": "node lib/changelog/generate-changelog-markdown-files.mjs",
     "build": "yarn build:packages && ember build --environment=production",
     "build:packages": "yarn workspace @hashicorp/design-system-components build",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint": "concurrently \"yarn:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"yarn:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "lint:css": "stylelint \"app/styles/**/*.scss\"",
-    "lint:css:fix": "npm-run-all \"lint:css --fix\"",
+    "lint:css:fix": "concurrently \"yarn:lint:css -- --fix\"",
```

```suggestion
    "lint:css:fix": "concurrently \"yarn:lint:css --fix\"",
```
[todo] Can you update this based on the findings that this command is not working in #2630?